packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -665,8 +665,11 @@ export class DashboardService {
         const { organizationUuid, projectUuid } = dashboard;
         if (
             user.ability.cannot(
-                'update',
-                subject('Dashboard', { organizationUuid, projectUuid }),
+                'manage',
+                subject('ScheduledDeliveries', {
```

Comment:
We should rename the method to be more explicit as it was initially intended to check dashboard access.

Reply:
done